pipelines/skylab/snap-atac/README.md

```diff
@@ -62,6 +62,7 @@ The pipeline inputs are detailed in the table below. You can test the workflow b
 | input_reference | File | Reference bundle that is generated with bwa-mk-index-wdl found [here](https://github.com/HumanCellAtlas/skylab/blob/master/library/accessory_workflows/build_bwa_reference/bwa-mk-index.wdl)|
 | genome_name | String | Name of the genomic reference (name that precedes the “.tar” in the input_reference) |
 | output_bam  | String  | Name for the output BAM |
+| bin_size_list  | String  | List of bin sizes used to generate cell-by-bin matrices; default is 10000 bp |
```

Comment:
Can users put in more than one number here? So they could do something like`10000 5000`?

Reply:
Thats a good question. @khajoue2 do you know?